unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UserContextMenu/Scripts/UserContextMenu.cs

```diff
@@ -201,7 +201,7 @@ private void Update()
 
     private void OnDisable()
     {
-        FriendsController.i.OnUpdateFriendship -= OnFriendActionUpdate;
+        Environment.i.serviceLocator.Get<IFriendsController>().OnUpdateFriendship -= OnFriendActionUpdate;
```

Comment:
you should allocate the `friendsController` instance so you avoid asking the service locator on each usage